tests/jballerina-integration-test/src/test/resources/grpc/src/grpcservices/03_bidirectional_streaming_service.bal

```diff
@@ -21,6 +21,9 @@ import ballerina/runtime;
 // Server endpoint configuration
 listener grpc:Listener ep3 = new (9093);
 
+map<grpc:Caller> consMap = {};
```

Comment:
Should we verify this change with the standard library team? I guess the examples would also have to change, maybe we can do that when revisiting the BBEs.

Can we also check if we can rename this variable? `consMap` sounds a bit like a const map.

Reply:
Yes, we have a similar sample in examples as well. will check it later.
This is a connection map. It's ok to change it to any meaningful name.